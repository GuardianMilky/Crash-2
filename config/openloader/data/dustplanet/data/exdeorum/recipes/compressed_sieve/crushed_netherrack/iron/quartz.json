{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/crushed_netherrack"},"mesh":"exdeorum:iron_mesh","result":"minecraft:quartz","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.11}}