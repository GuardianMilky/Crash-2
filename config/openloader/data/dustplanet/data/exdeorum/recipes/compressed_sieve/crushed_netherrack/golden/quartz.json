{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/crushed_netherrack"},"mesh":"exdeorum:golden_mesh","result":"minecraft:quartz","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.13}}